3-networks-hub-and-spoke/envs/shared/partner_interconnect.tf.example

```diff
@@ -18,7 +18,7 @@
 module "dns_hub_interconnect" {
   source = "../../modules/partner_interconnect"
 
-  vpc_name              = "net-dns"
+  vpc_name              = "vpc-net-dns"
```

Comment:
the infrastructure created by the module `"dns_hub_interconnect` is not necessary anymore.